### DISCUSSION TITLE
cannot run dp neighbor-stat

### DISCUSSION THREAD

**Discussion by LZYUCL** [deepmodeling/deepmd-kit#1607] [Q&A] [answered]
I cannot run dp neighbor-stat -s data -r 6.0 -t O H
Is there other ways to calculate the neighbor stat of the training data?
Thank you

**Comment by wanghan-iapcm** [comment, marked as answer]
Could you please report the version of DeePMD-kit and error message ? Thank you

**Reply by LZYUCL** [reply]
Thank you! 
I used DeePMD-kit v2.0.4, and the error message is below.
dp: error: argument command: invalid choice: 'neighbor-stat' (choose from 'config', 'transfer', 'train', 'freeze', 'test', 'compress', 'doc-train-input', 'model-devi', 'convert-from')

**Reply by njzjz** [reply]
It's a feature in [v2.1.0](https://github.com/deepmodeling/deepmd-kit/releases/tag/v2.1.0) though.

**Reply by LZYUCL** [reply]
Thank you! It is working now.

**Reply by liaonan2001** [reply]
Hello, I would like to ask how you solved

**Reply by LZYUCL** [reply]
Updata your deepmd-kit to v2.1.0

**Comment by liaonan2001** [comment]
您好，您的信件我已收到~This is an automatic reply, confirming that your e-mail was received.Thank you
